pkg/diff/diff.go

```diff
@@ -163,6 +154,26 @@ func (t *TableDiff) adjustConfig() {
 	if t.CheckThreadCount <= 0 {
 		t.CheckThreadCount = 4
 	}
+
+	return t.getTableInfo(ctx)
```

Comment:
I think calling this function in `Equal` following `adjustConfig` is better, so `adjustConfig` only adjusts the config.